### PR TITLE
Item form media visibility

### LINKED ISSUE #249: Set media visibility on item add/edit page
Users can set visibility on the media edit page. Users should also be able to set media visibility on the item add/edit page, in the media tab.

### PR DESCRIPTION
**Pull request by kimisgold**
Fix #249.

### CODE REVIEW

**Comment by jimsafley** [comment]
Users need to set visibility for new media, not just existing media. The default visibility for new media should be public.

**Comment by kimisgold** [comment]
Whoops, forgot that I hadn't pushed.

File: application/asset/js/resource-form.js [hunk 1]
```diff
@@ -97,12 +97,14 @@
             }
         });
 
+<<<<<<< HEAD
```
Comment (jimsafley) [inline application/asset/js/resource-form.js]: Looks like some merge conflict wasn't fully resolved, lines 100-101, 119.

**Comment by zerocrates** [comment]
There's also seemingly doubled-up commits for "set up media visibility.." and "explicitly index media.."

File: application/src/View/Helper/Media.php [hunk 2]
```diff
@@ -34,9 +34,14 @@ public function __construct(ServiceLocatorInterface $serviceLocator)
      */
     public function form($mediaType, array $options = array())
     {   
+        $mediaType = $this->getView()->escapeHtmlAttr($mediaType);
         $form = '<div class="media-field-wrapper">';
-        $form .= '<ul class="actions">
-            <li><a class="o-icon-delete remove-new-media-field" href="#"" title="Remove value" aria-label="Remove value"></a></li>
+        $form .= '<ul class="actions">';
+        if ($mediaType == 'upload') {
```
Comment (jimsafley) [inline application/src/View/Helper/Media.php]: What is the purpose of singling out the upload type here?

**Pull request closed without merging**